account-compression/README.md

```diff
@@ -0,0 +1,13 @@
+# Smart Contracts
+
+`gumdrop` will be managed by solana-labs 
```

Comment:
We probably need to significantly modify this README. I (and @jnwng ) will help you with dividing up some of these tasks.